Makefile

```diff
@@ -78,6 +78,7 @@ config.gypi: configure
 	fi
 
 install: all
+	-rm -rf '$PREFIX'/lib/node_modules/npm
```

Comment:
tbqh I was whipping this up while doing a handful of other things. I assumed the rimraf implementation would not be accepted and wanted some suggestions on next steps. I've updated with `${PREFIX)` so that it actually works, and will explore doing this in the python script